modules/nw-router-configuring-dual-stack.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-router-configuring-dual-stack_{context}"]
+= Configuring the {product-title} router for dual-stack support
+
+When you install a {product-title} cluster on bare metal infrastructure, you can configure the router for single-stack or dual-stack support.
```

Comment:
I'm not entirely sure how customers install dual stack clusters (and how that would differ from how we spin up internal dev dual-stack clusters). So might be best to specify that these instructions apply to dual-stack enabled bare metal openshift environments. Is there existing documentation explaining how to set up dual stack clusters (or is that on a need-to-know basis for specific customers?) If so, perhaps this section should link to those docs?

Reply:
I'm not entirely sure how the installation differs either, but I can ask. I'll look into putting this procedure in the installation book as well as the networking or linking to the installation docs. 

Adding the specification that these instructions apply to dual-stack enabled bare metal infrastructure. 

Also for consistency I'm changing "router" to "Ingress Controller"

Reply:
Actually, might be worth checking if dual-stack is a supported environment at all for 4.7? Im really not sure the GA status of that feature. I guess no harm in mentioning how it _would_ work for the router though?